### ISSUE TITLE
Implement nmod:desc for honorific pre-nominal titles: Mr., Dr., etc.

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#561]
- [x] EWT abbreviated titles and their expanded equivalents: ~https://universal.grew.fr/?custom=6769d90c72a6e~ https://universal.grew.fr/?custom=676acf880dbd0

Manually implementing the few that are not `flat` (many instances of "St." for "Saint" are `compound`). Will make a rule for the `flat` ones.

**Comment by nschneid** [comment]
- [ ] [EWT King, Queen, etc. as compound](https://universal.grew.fr/?custom=6769f392c3bfd)

**Comment by nschneid** [comment]
I reviewed [flat-initial PROPNs in GUM and EWT with filters for honorifics](https://universal.grew.fr/?custom=676a3c3aaaaf6). The lexical lists in the query can be used for a conversion rule.

In the above review, found a few GUM errors:
- [ ] PROPN should be ADJ: Emeritus (Professor), Golden, Chief (Justice)
- [ ] wrong structure (I would go with `compound`): Comic Corps, Rationalist Atheists, Senate Democrats, State Department, NASA Administrator

N.B. "Dean" and "Earl" are in principle ambiguous between titles and first names, but in the data they are only the latter.

**Comment by nschneid** [comment]
Mostly done with EWT but a few edeps from relative clauses to relativizer antecedents got deleted—need to investigate. (I think 2 new rules after invert headedness will solve this: enhanced relation constraints `#4~#1;#2.*#4`, `#4~#1;#4.*#2`, mark #4:edep as OLD and add `#4~#2`. Where `#1` is the honorific, now the head of `#2`, which begins the name part.)

**Comment by amir-zeldes** [comment]
Could you share your whole depedit script as a starting point for GUM too? We won't need the edeps part but the basic dependencies would benefit from it!

**Comment by AngledLuffa** [comment]
Oh, if such a script already exists, we could run that on ParTUT as well.  Could pass it along to @LarsAhrenberg for LinES

is updating ParTUT going to be a community effort?  seems like the author is still out there occasionally doing things, but only very occasionally

**Comment by nschneid** [comment]
https://github.com/UniversalDependencies/UD_English-EWT/blob/64538925ef30bf5b4af70039aff564afc8608159/not-to-release/tools/flat2nmoddesc.ini

But see the note at the end about nested cases requiring manual attention.

**Comment by nschneid** [comment]
> is updating ParTUT going to be a community effort?

I can answer annotation questions about other treebanks but in terms of proactive updates, I have my hands pretty well full maintaining EWT.

**Comment by amir-zeldes** [comment]
Yeah, same here - I have my hands full at the moment I'm afraid!

**Comment by AngledLuffa** [comment]
phrases such as `The playwright and critic George Bernard Shaw` do not get updated with `nmod:desc`?

and what is the correct arrangement for `Drs. Foo and Bar`?  `nmod:desc(Foo, Drs.)` and maybe an enhanced dependency `nmod:desc(Bar, Drs.)` if the treebank supports it, which is not the case for ParTUT anyway?

**Comment by nschneid** [comment]
> phrases such as `The playwright and critic George Bernard Shaw` do not get updated with `nmod:desc`?

No, the definite article indicates that there are 2 full nominal phrases (related by `appos`)

> and what is the correct arrangement for `Drs. Foo and Bar`? `nmod:desc(Foo, Drs.)`

Yes

> and maybe an enhanced dependency `nmod:desc(Bar, Drs.)` if the treebank supports it, which is not the case for ParTUT anyway?

Not sure about the enhanced dependency, especially since "Drs." is explicitly plural, and thus applies to the full coordination

**Comment by amir-zeldes** [comment]
So, quick question - do we use `nmod:desc` for "St. Petersburg"?

**Comment by nschneid** [comment]
I'd say yes, because the contribution of the "St." originally is as a title (of a name that got turned into a place name)

**Comment by amir-zeldes** [comment]
👍 Doing the same with St. Louis

**Comment by nschneid** [comment]
> I reviewed [flat-initial PROPNs in GUM and EWT and filtered out the honorifics](https://universal.grew.fr/?custom=676a55b37ff0d). The lexical lists in the query can be used for a conversion rule.
> 
> In the above review, found a few GUM errors:
> 
>   * [x]  PROPN should be ADJ: Emeritus (Professor), Golden (Gate), Chief (Justice)
>   * [x]  wrong structure (I would go with `compound`): Comic Corps, Rationalist Atheists, Senate Democrats, State Department, NASA Administrator, US President, Vice President
> 
> 
> N.B. "Dean" and "Earl" are in principle ambiguous between titles and first names, but in the data they are only the latter.

@amir-zeldes do these need fixing in GUM?

**Comment by AngledLuffa** [comment]
ParTUT:

"The Earl of Gloucester", "Edward De Vere, 17th earl of Oxford" - both NOUN, not PROPN?

What about "Henry Wriothesley, Earl of Southampton" - does the missing "the" make a difference?

PUD:

"chief executive" - you would have "chief" be ADJ?  same with "brand chief Herbert Diess"?

**Comment by nschneid** [comment]
"chief" is a NOUN in "brand chief". I would call it an ADJ in "chief executive", "chief justice", etc.

I see Queen/PROPN Elizabeth, and President/PROPN Bush, so I guess we're using PROPN in personal names for honorific titles not following a determiner.

Referring to an office, I see President/PROPN of the United States as well as president/NOUN of the United States (the tag seems to follow capitalization).

**Comment by amir-zeldes** [comment]
> @amir-zeldes do these need fixing in GUM?

GUM errors now fixed upstream, thanks!
